{{ cookiecutter.package_name|slugify }}/src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/__init__.py

```diff
@@ -1 +1,29 @@
 """{{ cookiecutter.package_name }} package."""
+{%- if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int %}
+
+import logging
+
+import coloredlogs
+
+
+def configure_root_logger() -> None:
+    """Configure the root logger."""
+    # Remove all handlers associated with the root logger object.
+    for handler in logging.root.handlers:
+        logging.root.removeHandler(handler)
+    # Add coloredlogs' coloured StreamHandler to the root logger.
+    coloredlogs.install(
```

Comment:
To simplify this function, I'd prefer to replace this call with `coloredlogs.install()` as I don't see anything in the configuration that we would really need to apply by default.

Reply:
Oki, changed to coloredlogs defaults.